pyvista/plotting/plotting.py

```diff
@@ -3273,15 +3273,11 @@ def add_point_labels(self, points, labels, italic=False, bold=True,
         self.remove_actor(f'{name}-labels', reset_camera=False)
 
         # add points
-        if show_points:
-            style = 'points'
-        else:
-            style = 'surface'
-        self.add_mesh(vtkpoints, style=style, color=point_color,
-                      point_size=point_size, name=f'{name}-points',
-                      pickable=pickable,
-                      render_points_as_spheres=render_points_as_spheres,
-                      reset_camera=reset_camera)
+        if show_points is True:
```

Comment:
Remove redundant `is True`:
```suggestion
        if show_points:
```